tests/gui/modules/progress-status.spec.js

```diff
@@ -22,7 +22,7 @@ describe('Browser: progressStatus', function () {
     })
 
     it('should report 0% if percentage == 0 but speed != 0', function () {
-      m.chai.expect(progressStatus.fromFlashState(this.state)).to.equal('0% Flashing')
+      m.chai.expect(progressStatus.fromFlashState(this.state, true)).to.equal('Flashing... 0%')
```

Comment:
Is it worth having a test for `.fromFlashState(this.state, false)` ? Or does `tests/gui/pages/main.spec.js` provide sufficient coverage for that?